packages/playground/src/webapp/FreonLayout.svelte

```diff
@@ -39,8 +39,8 @@
 
```

Comment:
Looks good, but can you please add something to the documentation, both website and readme files: a short remark how to use this or simply a todo specifying that this feature needs to be documented. Nothing fancy, just a simple reminder in the right places.

Reply:
Done